azerothcore/azerothcore-wotlk
# fix(DB/SAI) Add cosmetic script to Rousch

## Changes Proposed:
-  Add SAI to npc Rousch. Purely cosmetic.

## Issues Addressed:

- Closes 
- #6912
- https://github.com/chromiecraft/chromiecraft/issues/2199

## SOURCE:
Video in issue.

## Tests Performed:
- Tested SAI works like video

## How to Test the Changes:
1. .go c 14809 and observe Rousch's behaviour.
2. Run SQL
3. .reload smart_scripts
